decidim-proposals/lib/decidim/proposals/component.rb

```diff
@@ -265,7 +265,20 @@
         visibility: "all"
       ) do
         proposal = Decidim::Proposals::Proposal.new(params)
+
         proposal.add_coauthor(participatory_space.organization)
+
+        coauthor = case n
+                   when 0
+                     Decidim::User.where(decidim_organization_id: participatory_space.decidim_organization_id).order(Arel.sql("RANDOM()")).first
+
+                   when 1
+                     Decidim::UserGroup.where(decidim_organization_id: participatory_space.decidim_organization_id).order(Arel.sql("RANDOM()")).first
+                   when 2
+                     Decidim::Meetings::Meeting.where(component: component).order(Arel.sql("RANDOM()")).first
+                     participatory_space.organization
```

Comment:
```suggestion
                   else
                     participatory_space.organization
```
Missing an else statement?